praw/models/reddit/collections.py

```diff
@@ -165,6 +165,27 @@ def update_description(self, description: str):
             data={"collection_id": self.collection_id, "description": description},
         )
 
+    def update_display_layout(self, display_layout: str):
+        """Update the collection's display layout.
+
+        :param display_layout: Either `TIMELINE` or `GALLERY`.
+
+        Example usage:
+
+        .. code-block:: python
+
+            collection = reddit.subreddit("SUBREDDIT").collections("some_uuid")
```

Comment:
```suggestion
            collection = reddit.subreddit("test").collections("some_uuid")
```
Sometime in the future all the subreddits used in the code examples will become standardized to r/test.